.github/actions/package-review/action.yml

```diff
@@ -0,0 +1,29 @@
+# Relevant documentation:
+# https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action
+# https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions
+
+name: Package Review Action
+description: 'Run all examples and tests in a package'
+
+inputs:
+  package:
+    description: 'Package name'
+    required: true
+    default: 'FirstPackage'
+outputs:
+  numExampleErrors:
+    description: 'Number of errors occured running examples'
+  numTestErrors:
+    description: 'Number of errors occured running tests'
+runs:
+  using: 'docker'
+  image: 'docker://mahrud/macaulay2:v1.16.0.1'
```

Comment:
By the way, if this turns out to be used often, we'll want a "Macaulay2" docker account.

Reply:
It might already be [taken](https://hub.docker.com/u/macaulay2), unless @mikestillman owns that. Regardless, github has its own docker registry. I just didn't use it because it requires an authentication step.

Reply:
Or could it be Lars Kastner?  We should find out.